app/src/main/java/com/toolslab/reviewme/MainActivity.kt

```diff
@@ -2,11 +2,16 @@ package com.toolslab.reviewme
 
 import androidx.appcompat.app.AppCompatActivity
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/errysuprayogi/ReviewMe/blob/fcf709e0ffecb4c1f654fac82fcf94981125d9a8/app/src/main/java/com/toolslab/reviewme/MainActivity.kt#L3"></span>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end</td>
    </tr>
  </tbody>
</table>